index.js

```diff
@@ -8,8 +8,51 @@ const http = require('http'),
         The time is: ${rightNow}
       </p>
     </body>
+ 
+      <body>
+   <div style="position:  absolute; bottom: 5px; background-color: white">
```

Comment:
pull out the styles for now :-)

Achieve positioning by placing the content in the correct order in the document